src/cli/index.ts

```diff
@@ -8,19 +8,23 @@ import { ArgumentsWithAliases, argumentsWithAliases, CliOptions } from "./cli-ar
 import { spinner } from "./spinner";
 
 export async function cli(args: string[]): Promise<void> {
+  try {
+    await runCli(args);
+  } catch (e) {
+    spinner.fail(e?.message ?? e ?? "Unknown error");
+  }
+}
+
+async function runCli(args: string[]) {

```

Comment:
Not a huuuge fan of using the verb 'run' to differentiate it from the exported 'cli' method.

Perhaps this method could be called 'cli' and then the exported method could be something more generic like 'main'? Thinking along the lines of comparing it to a C# entry method since 'main' would be the method the binary would hook into (I think)?